wrappers/python/zxing.cpp

```diff
@@ -47,6 +51,30 @@ Result decode(const Image& image, BarcodeFormat format, bool fastMode, bool tryR
 	return decode(image, {format}, fastMode, tryRotate, hybridBinarizer);
 }
 
+Image encode(std::string text, int width, int height, BarcodeFormat format, int margin, int eccLevel) {
+	MultiFormatWriter writer(format);
+	if (margin >= 0)
+		writer.setMargin(margin);
+	if (eccLevel >= 0)
+		writer.setEccLevel(eccLevel);
+
+	auto bitmap = writer.encode(TextUtfEncoding::FromUtf8(text), width, height).toByteMatrix();
+
+	auto bitmapWidth = bitmap.width();
+	auto bitmapHeight = bitmap.height();
+
+	py::array_t<uint8_t> result = py::array_t<uint8_t>(bitmapWidth * bitmapHeight * sizeof(uint8_t));
+	auto buf3 = result.request();
+	int8_t *ptr3 = (int8_t *) buf3.ptr;
```

Comment:
Since your array is explicitly containing `uint8_t`s, why use a `int8_t` here? Also, according to the pybind11 doc, it seems a more canonical way would be something like:
```
    auto r = x.mutable_unchecked<2>();
    for (ssize_t i = 0; i < r.shape(0); i++)
        for (ssize_t j = 0; j < r.shape(1); j++)
                r(i, j) = ...;
```